xml/ha_config_cli.xml

```diff
@@ -356,9 +387,187 @@ Operations' defaults (advisory minimum):
    </tip>
   </sect2>
 
+  <sect2 xml:id="sec.ha.manual_config.clusterscripts">
+    <title>Using &crmsh;'s Cluster Scripts</title>
+    <remark>toms 2015-10-14: FATE#318211</remark>
+    <para>One big task of configuring and managing a cluster is to collect
+      all information of its nodes and deploying the changes. Performing the
+      same task manually on different nodes can be error-prone. To make this
+      process simpler, easier, repetitive, and more consistent, the &crmsh; cluster
+      scripts were invented.
+    </para>
+
+    <para>The &crmsh; cluster scripts have the following objectives:</para>
+    <itemizedlist>
+      <listitem>
+        <para>Install software which is required for a specific task.</para>
+      </listitem>
+      <listitem>
+        <para>Create or modify any configuration files.</para>
+      </listitem>
+      <listitem>
+        <para>Collect information and report potential problems with the
+          cluster.</para>
+      </listitem>
+      <listitem>
+        <para>Deploy the changes to all nodes.</para>
+      </listitem>
+    </itemizedlist>
+
+    <para>It is not intented to replace other tools for managing clusters,
+      rather, than provide an integrated way to perform the above tasks across
+      the cluster. Find detailed information at <link
+        xlink:href="http://crmsh.github.io/scripts/"
+        xlink:show="new"/>.
+    </para>
+
+    <sect3 xml:id="sec.ha.manual_config.clusterscripts.usage">
+      <title>Usage</title>
+      <para>To get a list of all available cluster scripts, run:</para>
+      <screen>&prompt.root;<command>crm</command> script list</screen>
+
+      <para>Each cluster script contains more information than just a
```

Comment:
Maybe shorten to: To view the components of a script, use the <command>show</command> command and the name of the cluster script, for example: